cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -141,17 +150,25 @@ private def artifactCleanup(int months, String[] repos, log, paceTimeMS, dryRun
                 cntNoDeletePermissions++
             }
             if (dryRun) {
-                log.info "Found $it, $cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
-                log.info "\t==> currentUser: ${security.currentUser().getUsername()}"
-                log.info "\t==> canDelete: ${security.canDelete(it)}"
-
+                if (checkName(keepRelease, releaseRegex, it)) {
+                    log.info "Found $it, $cntFoundArtifacts/$artifactsCleanedUp.size total $bytesFound bytes"
```

Comment:
The 'Found' and 'currentUser' log calls can be moved out of the if/else block (DRY)

Reply:
Yes, I am doing refactoring of this part